spec/fake_braintree/customer_spec.rb

```diff
@@ -54,6 +54,32 @@
     expect(result).to be_success
   end
 
+  it 'creates a credit card from payment method nonce in credit card hash' do
+    nonce = FakeBraintree::PaymentMethod.tokenize_card({
+      number: TEST_CC_NUMBER,
+      expiration_date: '04/2016'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.